app/controllers/uploads_controller.rb

```diff
@@ -71,7 +71,11 @@ def current_stream
   end
 
   def create_params
-    upload_params.merge(stream_id: current_stream.id)
+    upload_params.merge(
+      stream_id: current_stream.id,
+      user_id: current_user&.id,
+      ip_address: request.env['HTTP_X_FORWARDED_FOR'] || request.remote_ip
```

Comment:
Is it also possible to track which token was used, if any?

Reply:
Sure thing!